### PR TITLE
5th try

### PR DESCRIPTION
**Pull request by leealbert95**

### CODE REVIEW

**Comment by leealbert95** [comment]
@2000prcs  @jennqiao  @sujithavikram  I got Travis CI to work!

**Comment by sujithavikram** [comment]
@leealbert95 Hey Albert it looks good.. great job!!

**Comment by 2000prcs** [comment]
@leealbert95 Looks good to me!

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const path = require('path');
+
+const app = express();
+const port = process.env.PORT || 3000;
```
Comment (jennqiao) [inline server/index.js:7]: Sujitha is using port 3000 - perhaps switch to another one?

**Pull request merged**